.github/workflows/t-bytecode-compare.yml

```diff
@@ -0,0 +1,94 @@
+name: Release bytecode
+
+on: pull_request
+
+jobs:
+   generate:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        include:
+          - platform: emscripten-wasm32
+            os: ubuntu-latest
+
+          - platform: emscripten-asmjs
+            os: ubuntu-latest
+
+          - platform: linux-amd64
+            os: ubuntu-latest
+
+          - platform: macosx-amd64
+            os: macos-latest
+
+          # TODO: Run checks on Windows binaries too
+
+    env:
+      PLATFORM: ${{ matrix.platform }}
+
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          # Use the latest minor release of Python 3. prepare_report.py requires Python >= 3.7
+          python-version: '3.x'
+
+      - uses: actions/checkout@v2
+        with:
+          repository: 'ethereum/solidity'
+          path: 'solidity/'
+          # bytecode_reports_for_modified_binaries.sh requires access to a working copy with full history
+          fetch-depth: 0
+          # TMP: This job uses scripts from my yet unmerged PR in Solidity repo
+          ref: solc-bin-bytecode-comparison-pr-check-scripts
```

Comment:
Note: this and the one below need to be changed before merging.

Reply:
Right. I'll switch it back to draft in the meantime just in case.

Reply:
Done.